crates/lakekeeper/src/service/tasks/mod.rs

```diff
@@ -96,14 +100,18 @@ pub enum TaskEntity {
 pub enum TaskEntityNamed {
     Table(TableNamed),
     View(ViewNamed),
+    Project(ProjectId),
+    Warehouse(WarehouseId),
```

Comment:
This is called `Named` yet we don't have names for Project & Warehouse. This is quite unexpected. Something feels off.